assets/wire/utils.js

```diff
@@ -49,6 +49,35 @@ function getRelatedItemsByType(item, type) {
     return item.type === type ? [item] : Object.values(get(item, 'associations', {}) || {}).filter((assoc) => get(assoc, 'type') === type);
 }
 
+export function getFeatureMedia(item) {
+    if (['picture', 'video', 'audio'].includes(item.type)) {
+        return item;
+    }
+
+    const featured = get(item, 'associations.featuremedia');
+
+    if (featured != null && ['picture', 'video', 'audio'].includes(featured.type)) {
+        return featured;
+    }
+
+    return getBodyPicture(item);
+}
+
+export function getOtherMedia(item) {
+    if (['picture', 'video', 'audio'].includes(item.type)) {
+        return null;
+    }
+
+    return Object.keys(item.associations || {})
+        .filter((key) => (
+            !key.startsWith('editor_') &&
```

Comment:
what the meaning of `key.startsWith('editor_')`?

Reply:
If you look at `apps.archive.archive.update_associations` in the backend (https://github.com/superdesk/superdesk-core/blob/develop/apps/archive/archive.py#L212), media items from `fields_meta.body_html.draftjsState.body_html` metadata are stored in `assoications` under the `editor_` prefix